docs/reference/supported-languages.md

```diff
@@ -10,9 +10,10 @@ Bearer currently supports Ruby and JavaScript codebase.
 | Languages               | Supported |
 | ----------------------- | --------- |
 | Ruby                    | ✅         |
-| JavaScript / TypeScript | ✅         |
+| JavaScript              | ✅         |
+| TypeScript              | ✅         |
```

Comment:
> Is there a reason to separate them? We treat their rules as one, so it may be confusing to split them up elsewhere.

I think it clarifies things, because this is not the last one where we have this, for e.g Java and Scala etc.